live-assignments/anagram/answer/app/js/anagram.js

```diff
@@ -0,0 +1,16 @@
+function AnagramChecker () { }
+AnagramChecker.prototype.checkAnagrams = function(word1, word2) {
+	if (word1 == null || word2 == null) {
+		return false;
+	}
+	return this.prepareWord(word1) === this.prepareWord(word2);
+};
+AnagramChecker.prototype.prepareWord = function(word) {
+	return word.split('').sort().reduce(function(a, b) { return a + b; });
```

Comment:
You don't  have to change it if you don't want, but this does the same and I think is a little easier to read:

```suggestion
	return word.split('').sort().join('');
```

Reply:
Had a feeling there might be a better way, thanks.